core/domain/email_manager.py

```diff
@@ -1155,10 +1275,351 @@ def send_mail_to_notify_users_to_review(user_id, category):
+    """
+    email_subject = (
+        NOTIFY_ADMINS_SUGGESTIONS_WAITING_FOR_REVIEW_EMAIL_INFO[
+            'email_subject'])
+    email_body_template = (
+        NOTIFY_ADMINS_SUGGESTIONS_WAITING_FOR_REVIEW_EMAIL_INFO[
+            'email_body_template'])
+
+    if not feconf.CAN_SEND_EMAILS:
+        log_new_error('This app cannot send emails to users.')
+        return
+
+    if not (
+            config_domain
+            .NOTIFY_ADMINS_SUGGESTIONS_WAITING_TOO_LONG_IS_ENABLED.value):
+        log_new_error(
+            'The "notify_admins_suggestions_waiting_too_long" property '
+            'must be enabled on the admin config page in order to send '
+            'admins the emails.'
+        )
+        return
+
+    if not reviewable_suggestion_email_infos:
+        log_new_error(
+            'There were no Contributor Dashboard suggestions that were waiting '
+            'too long for a review.')
+        return
+
+    if not admin_ids:
+        log_new_error(
+            'No admins to notify that Contributor Dashboard suggestions have '
+            'waited too long for a review.')
+        return
+
+    suggestions_to_notify_admin_html_strings = []
+    # Get the html for the list of suggestions that have been waiting too long
+    # for a review.
+    for reviewable_suggestion_email_info in reviewable_suggestion_email_infos:
+        suggestions_to_notify_admin_html_strings.append(
+            _create_html_for_reviewable_suggestion_email_info(
+                reviewable_suggestion_email_info))
+
+    suggestions_to_notify_admin_html = ''.join(
+        suggestions_to_notify_admin_html_strings)
+
+    # Get the emails and usernames of the admins.
+    admin_user_settings = user_services.get_users_settings(admin_ids)
+    admin_usernames, admin_emails = list(python_utils.ZIP(*[
+        (admin_user_setting.username, admin_user_setting.email)
+        if admin_user_setting is not None else (None, None)
+        for admin_user_setting in admin_user_settings
+    ]))
+
+    for index, admin_id in enumerate(admin_ids):
+        if not admin_emails[index]:
+            log_new_error(
+                'There was no email for the given admin id: %s.' % admin_id)
+            continue
+        else:
+            email_body = email_body_template % (
+                admin_usernames[index], feconf.OPPIA_SITE_URL,
+                feconf.CONTRIBUTOR_DASHBOARD_URL,
+                suggestion_models.SUGGESTION_REVIEW_WAIT_TIME_THRESHOLD_IN_DAYS,
+                feconf.OPPIA_SITE_URL, feconf.ADMIN_URL,
+                suggestions_to_notify_admin_html)
+
+            _send_email(
+                admin_id, feconf.SYSTEM_COMMITTER_ID,
+                feconf.EMAIL_INTENT_ADDRESS_CONTRIBUTOR_DASHBOARD_SUGGESTIONS,
+                email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS,
+                recipient_email=admin_emails[index])
+
+
+def send_mail_to_notify_admins_reviewers_needed(
```

Comment:
Can you have the end of this be "that_reviewers_are_needed"?

Reply:
if it isn't too long for the line count then yes (I tried to make the names short since there were nested swaps in the tests)